### ISSUE TITLE
Firefox 4.X: Logging into secure page, NVDA doesn't have focus in virtual document

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1438]
**Reported by kevinchao89 on 2011-03-29 01:42**
Firefox 4.2a1pre (2011-03-28) 
Firefox 4 final. 
NVDA snapshot 4138
NVDA 2011.1

Visit http://GoSendero.com 
Account ID: demo2011
Password: sendero
TAB to login button and ENTER. 
Actual: ARROWS and Elements list don't do anything. TAB and SHIFT+TAB goes between remember password submenu and remember password button.  
Must ALT+TAB to a different application, ALT+TAB back to Firefox, then TAB and SHIFT+TAB. At which point, NVDA focus is in document and arrows, elements list, and tab work.  
Expected:No need for work-around. Arrows, Elements list, and TAB to work. 

This is not a unique issue to this specific page, but occurs on many pages, which require login. I use three other sites very frequently, which have this same very issue.

**Comment by ehollig** [comment]
This is not the case anymore with Firefox 54.0.1.

**Comment by leonardder** [comment]
Closing as works for me.
